### DISCUSSION TITLE
Is there any rule for the sizing values used in Bootstrap SCSS?

### DISCUSSION THREAD

**Discussion by MohamadSalman11** [twbs/bootstrap#40898] [Q&A] [answered]
I often see in SCSS files that sizing values such as padding-top: 1.25rem and margin: 0.75rem etc. are commonly used. Should I always follow the rule of using increments of 0.25, or can I use any value like padding: 0.19rem? Is there a resource that explains such guidelines?

**Comment by julien-deramond** [comment, marked as answer]
I'm not sure if there are explicit rules documented somewhere, but it’s primarily tied to the spacing scale you're using. Common values like `0.25rem`, `0.75rem`, and `1.25rem` are based on a `16px` font size base, so, for instance, `0.25 * 16` gives you `4px`. If you use something like `0.19rem`, that would equal `3.04px`, which isn’t ideal because non-integer pixel values can lead to inconsistencies or rounding issues in rendering. 
On top of that, the choice between using `rem`, `px`, or `em` can also depend on how you want the layout to behave when users zoom in or out. It depends on the use cases and renderings.
